### DISCUSSION TITLE
vc-overlay-echarts 标签在三维球上添加 Echart 迁徙图点击问题

### DISCUSSION THREAD

**Discussion by avllin** [zouyaoji/vue-cesium#551] [Q&A] [answered]
使用 vc-overlay-echarts 标签在三维球上添加 Echart 迁徙图效果，如果我想在点击获取当前迁徙的信息，该怎么做？
在这个标签上没有提供事件方法。

**Comment by zouyaoji** [comment, marked as answer]
这个无解哈。

https://github.com/zouyaoji/vue-cesium/discussions/171
